src/decode.h

```diff
@@ -890,6 +890,7 @@ void CaptureStatsSetup(ThreadVars *tv, CaptureStats *s);
 enum DecodeTunnelProto {
     DECODE_TUNNEL_ETHERNET,
     DECODE_TUNNEL_ERSPAN,
+    DECODE_TUNNEL_ERSPAN_TYPEI,
```

Comment:
So is our default ERSPAN actually type 2? Maybe we can just name these `DECODE_TUNNEL_ERSPAN2` and `DECODE_TUNNEL_ERSPAN1` then?